examples/android/helloworld/app/CMakeLists.txt

```diff
@@ -113,6 +117,7 @@ target_include_directories(grpc-helloworld
 target_link_libraries(grpc-helloworld
   libgrpc++
```

Comment:
replace by `grpc::libgrpc++` and get rid of all the explicit dependencies.
I think the `grpc::`  is the "project name" from here: https://github.com/grpc/grpc/blob/d9ec9068c7e7b38379229ea10b6b55d43e8dc549/CMakeLists.txt#L31

Reply:
Actually, with a well-defined helloworld_proto_lib  (which seems to be well defined), no extra dependencies should be necesssary.